src/problem.cpp

```diff
@@ -490,14 +490,15 @@ Problem::FindSolutionsImpl(Handle& handle,
     {
         if(!solver->IsApplicable(ctx, problem_description))
         {
+            MIOPEN_LOG_I2("solver " << solver->SolverDbId() << "is not applicable for the given problem description");
```

Comment:
Space before "is" is missing. Please look there and borrow message format for it: https://github.com/ROCm/MIOpen/blob/cee33a7aaa3dbbe850d2580a72fff8774ddbc131/src/include/miopen/find_solution.hpp#L209

Reply:
[resolved]